Modules/_cursesmodule.c

```diff
@@ -3815,8 +3815,11 @@ static int
 _curses_napms_impl(PyObject *module, int ms)
 /*[clinic end generated code: output=5f292a6a724491bd input=c6d6e01f2f1df9f7]*/
 {
-    PyCursesStatefulInitialised(module);
-
+    if (!_PyCursesStatefulCheckFunction(module,
```

Comment:
Hmmmm. It looks good, I admit. But the difference is only in `return 0` and `return -1`. It seems a bit cumbersome.

Reply:
It is necessary. Returning -1 with AC when an exception will re-raise this exception. Returning 0 in the other cases is to allow conversion to a NULL pointer when the return type is a `PyObject *`.